### DISCUSSION TITLE
ERROR: sync wait :handler: Websocket connection closed Problem

### DISCUSSION THREAD

**Discussion by yzliew38** [filecoin-project/lotus#8185] [Build and Sync] [answered]
After i key in the command 

lotus-miner init --owner=<address>  --worker=<address> --no-local-storage

it took me 1 days to download 3 files total around 100GIB +
keep disconnect and reconnect download till it done.

after that it show a command 

ERROR:sync wait :handler :websocket connection closed

May i know any problem and the solution ?

**Comment by LexLuthr** [comment, marked as answer]
This sounds like a connection issue. Please check your network and proxy if set.

**Comment by LexLuthr** [comment]
I am archiving this discussion as there has been no new input from the author for more than 7 days.
